uswds/public-sans
# Public-sans: Update Node LTS

# Summary

**Set node 18 as the default.** Updated `.nvmrc` file from 16→18, which is the current LTS version. 

## Breaking change

This is not a breaking change.

## Related issue

Closes #281 . 

## Related pull requests

Related to: 
uswds/uswds-site#2304
uswds/uswds#5560

## Preview link

[Public-sans →](https://federalist-514d1714-e827-44f8-a29f-3ec1837bf619.sites.pages.cloud.gov/preview/uswds/public-sans/cm-update-node-lts-18/)

## Problem statement

Support for node 16 is going to stop on CloudGov pages.

## Solution

1. Update `.nvmrc` to 18.
2. Run `npm audit fix`.
3. Update ruby gems.

## Testing and review

1. Run `npm install` and confirm no errors.
2. Run `npm run build` and confirm no errors.
3. Run `npm run start` and confirm no errors.
4. Run `npm run serve` and confirm site builds as expected.


### Checklist

- [ ] No errors when installing, building, starting, or serving the site
- [ ] No regressions on public-sans demo site

## Gem updates

_Indented gems are sub dependencies_


Gem name | Previous version | New version
-- | -- | --
addressable | 2.8.0 | 2.8.5
public_suffix | ≥ 2.0.2, < 5.0 | ≥ 2.0.2, < 6.0
concurrent-ruby | 1.1.10 | 1.2.2
ffi | 1.15.5 | 1.16.3
google-protobuf | - | 3.24.4
i18n | 1.10.0 | 1.14.1
jekyll | 4.2.2 | 4.3.2
| - jekyll-sass-converter | ~> 2.0 | ≥ 2.0, <4.0 | 
| - kramdown | ~>2.3 | ~> 2.3, ≥ 2.3.1 |
| - mercenary | ~> 0.4.0 | ≥ 3.0, < 0.5 |
| - rouge | ~> 3.0 | ≥ 3.2, < 5.0 |
| - terminal-table | ~> 2.0 | ≥ 1.8, <4.0 |
| - webrick | - | ~> 1.7 |
jekyll-sass-converter | 2.2.0 | 3.0.0
| - sassc | > 2.0.1, < 0.33 | - |
| - sass-embedded | - | ~> 1.54 |
liquid | 4.0.3 | 4.0.4
listen | 3.7.1 | 3.8.0
public_suffix | 4.0.7 | 5.0.3
rack | 2.2.6.2 | 2.2.8
rb-fsevent | 0.11.1 | 0.11.2
rexml | 3.2.5 | 3.2.6
rouge | 3.28.0 | 4.1.3
sassc | 2.4.0 | -
| - ffi | ~> 1.69 | - |
sass-embedded | - | 1.69.4
| - google-protobuf | - | ~> 3.23 |
| - rake | - | ≥ 13.0.0 |
terminal-table | 2.0.0 | 3.0.2
| - unicode-display_width | ~> 1.1, ≥ 1.1.1 | ≥ 1.1.1, < 3 |
thin | 1.8.1 | 1.8.2
unicode-display_width | 1.8.0 | 2.5.0
webrick | 1.7.0 | 1.8.1
